clarity/src/vm/mod.rs

```diff
@@ -428,8 +430,18 @@ pub fn execute_with_parameters(
     let mut contract_context = ContractContext::new(contract_id.clone(), clarity_version);
     let mut marf = MemoryBackingStore::new();
     let conn = marf.as_clarity_db();
-    let mut global_context =
-        GlobalContext::new(use_mainnet, conn, LimitedCostTracker::new_free(), epoch);
+    let chain_id = if use_mainnet {
+        CHAIN_ID_MAINNET
+    } else {
+        CHAIN_ID_TESTNET
+    };
```

Comment:
Can use `test_only_mainnet_to_chain_id` here (and remove const imports).

Reply:
Fixed